### PR TITLE
llzip

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/linkedList/llzip/ll-zip.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+    if (currentNodeTwo.next === null) {
+      currentNodeOne = currentNodeOne.next;
+    }
+    currentNodeOne = currentNodeOne.next;
+    currentNodeTwo = currentNodeTwo.next;
+  }
+  return givenListOne;
+}
+
```
Comment (adrianhuebner) [inline javascript/linkedList/llzip/ll-zip.js:28]: Looks wonderful. For tests think about creating two linked lists, and then just zipping them together, you could use your string method to test what you are getting back if you wanted.
